src/automation/README.md

```diff
@@ -31,7 +31,7 @@ az automation runbook create \
     --location westus
 ```
 
-##### Set runbook content (Download [AzureAutomationTutorial.ps1](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-automation/scripts/AzureAutomationTutorial.ps1) as an example)
+##### Set runbook content (Download [AzureAutomationTutorial.ps1](https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.automation/101-automation/scripts/AzureAutomationTutorial.ps1) as an example)
```

Comment:
May I ask could you use the `main` branch instead of `master` branch?
In order to avoid racial discrimination and eliminate sensitive words, we have an announcement [link](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-sdk%2Fissues%2F2745&data=04%7C01%7CZhou.Xing%40microsoft.com%7C2468b047df534406946908da0767c04e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637830439490159422%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=6eXwr7%2FOMjaViZ%2FryjQ08IBALSKCX7Pgl8ILFdOCq3w%3D&reserved=0) to rename the branch name of GitHub.

Reply:
@zhoxing-ms The azure-quickstart-templates doesn't have a main branch. It only has master branch.

![image](https://user-images.githubusercontent.com/57353862/173294255-6c9d5851-8b9f-4a7b-a24b-a18057938824.png)

Reply:
Yes, I noticed before. I mean could you create a main branch for it?

Reply:
unfortunately no.